projects/ontimize-web-ngx/src/lib/components/container/o-container-collapsible-component.class.ts

```diff
@@ -90,30 +90,12 @@ export class OContainerCollapsibleComponent extends OContainerComponent implemen
     }
   }
 
-  protected updateHeightExpansionPanelContent(): void {
-    const exPanelHeader = this._titleEl ? (this._titleEl as any)._element.nativeElement : null;
-    const exPanelContent: HTMLElement = this._containerCollapsibleRef ? this._containerCollapsibleRef.nativeElement.querySelector('.o-container-scroll') : null;
-    const parentHeight = exPanelHeader.parentNode ? exPanelHeader.parentNode.offsetHeight : null;
-
-    const height = (OContainerComponent.APPEARANCE_OUTLINE === this.appearance) ? parentHeight : (parentHeight - exPanelHeader.offsetHeight);
-    if (height > 0) {
-      exPanelContent.style.height = height + 'px';
+  updateInnerHeight(height: number): void {
+    if (this.containerContent) {
```

Comment:
It would be great checking that _number_ is really a number and valid to assign into style.height and instead assign 'null' maybe inherit ou auto?

Reply:
This method is called from the component template with values `null` or `0`. If the developer uses this method, the parameter is already secured with type number.